keyboards/jidohun/km113/info.json

```
@@ -0,0 +1,272 @@
+            {"matrix": [2, 11], "x": 164, "y": 26, "flags": 4},
+            {"matrix": [2, 10], "x": 149, "y": 26, "flags": 4},
+            {"matrix": [2, 9], "x": 134, "y": 26, "flags": 4},
+            {"matrix": [2, 8], "x": 119, "y": 26, "flags": 4},
+            {"matrix": [2, 7], "x": 105, "y": 26, "flags": 4},
+            {"matrix": [2, 6], "x": 90, "y": 26, "flags": 4},
+            {"matrix": [2, 5], "x": 75, "y": 26, "flags": 4},
+            {"matrix": [2, 4], "x": 60, "y": 26, "flags": 4},
+            {"matrix": [2, 3], "x": 45, "y": 26, "flags": 4},
+            {"matrix": [2, 2], "x": 30, "y": 26, "flags": 4},
+            {"matrix": [2, 1], "x": 15, "y": 26, "flags": 4},
+            {"matrix": [1, 1], "x": 15, "y": 13, "flags": 4},
+            {"matrix": [1, 2], "x": 30, "y": 13, "flags": 4},
+            {"matrix": [1, 3], "x": 45, "y": 13, "flags": 4},
+            {"matrix": [1, 4], "x": 60, "y": 13, "flags": 4},
+            {"matrix": [1, 5], "x": 75, "y": 13, "flags": 4},
+            {"matrix": [1, 6], "x": 90, "y": 13, "flags": 4},
+            {"matrix": [1, 7], "x": 105, "y": 13, "flags": 4},
+            {"matrix": [1, 8], "x": 119, "y": 13, "flags": 4},
+            {"matrix": [1, 9], "x": 134, "y": 13, "flags": 4},
+            {"matrix": [1, 10], "x": 149, "y": 13, "flags": 4},
+            {"matrix": [1, 11], "x": 164, "y": 13, "flags": 4},
+            {"matrix": [1, 12], "x": 179, "y": 13, "flags": 4},
+            {"matrix": [1, 13], "x": 194, "y": 13, "flags": 4},
+            {"matrix": [0, 13], "x": 194, "y": 0, "flags": 4},
+            {"matrix": [0, 12], "x": 179, "y": 0, "flags": 4},
+            {"matrix": [0, 11], "x": 164, "y": 0, "flags": 4},
+            {"matrix": [0, 10], "x": 149, "y": 0, "flags": 4},
+            {"matrix": [0, 9], "x": 134, "y": 0, "flags": 4},
+            {"matrix": [0, 8], "x": 119, "y": 0, "flags": 4},
+            {"matrix": [0, 7], "x": 105, "y": 0, "flags": 4},
+            {"matrix": [0, 6], "x": 90, "y": 0, "flags": 4},
+            {"matrix": [0, 5], "x": 75, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 60, "y": 0, "flags": 4},
+            {"matrix": [0, 3], "x": 45, "y": 0, "flags": 4},
+            {"matrix": [0, 2], "x": 30, "y": 0, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 13, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 26, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 38, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 51, "flags": 4},
+            {"matrix": [5, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [5, 1], "x": 15, "y": 64, "flags": 4},
+            {"matrix": [5, 2], "x": 30, "y": 64, "flags": 4},
+            {"matrix": [5, 5], "x": 75, "y": 64, "flags": 4}
+        ],
+        "max_brightness": 108,
+        "speed_steps": 64,
+        "val_steps": 27
```

These values are poor choices as default values.

```suggestion
        "max_brightness": 108
```